sdk/storage/Azure.Storage.Blobs/src/Models/BlobUploadFromUriOptions.cs

```diff
@@ -53,5 +53,13 @@ public class BlobUploadFromUriOptions
         /// Block Blob.
         /// </summary>
         public AccessTier? AccessTier { get; set; }
+
+        /// <summary>
+        /// Optional. An MD5 hash of the block content. This hash is used to verify the integrity of the block during
+        /// transport.When this header is specified, the storage service compares the hash of the content that has arrived
```

Comment:
nit: this should say "blob" not "block". or maybe we should just say "content" as source can be anything.
this doc should be fixed in java as well.